src/components/MapContainer.svelte

```diff
@@ -9,14 +9,14 @@
 	import { onMount } from 'svelte';
 	import { MapSource } from './../config/mapSource.js';
 
-	let apiKey = env.GOOGLE_MAPS_API_KEY;
+	let apiKey = env.PUBLIC_OBA_GOOGLE_MAPS_API_KEY;
 	let { handleStopMarkerSelect, mapProvider = $bindable(), ...restProps } = $props();
 
 	onMount(() => {
 		if (PUBLIC_OBA_MAP_PROVIDER === MapSource.Google) {
-			mapProvider = new GoogleMapProvider(apiKey);
+			mapProvider = new GoogleMapProvider(apiKey, handleStopMarkerSelect);
```

Comment:
Felt slightly unholy to be passing this function to the map providers, but not sure how else to get the Stop Modal to show up.

Reply:
This looks pretty idiomatic to me, actually. 👍